mysql/assets/dashboards/overview-screenboard.json

```diff
@@ -1,939 +1,1243 @@
+              "show_date_column": true,
+              "show_message_column": true,
+              "message_display": "expanded-md"
+            },
+            "layout": {
+              "x": 0,
+              "y": 0,
+              "width": 12,
+              "height": 10
+            }
           }
         ],
-        "title": "Failed MySQL Connects",
-        "title_size": "16",
-        "title_align": "left",
-        "precision": 2
-      },
-      "layout": {
-        "x": 54,
-        "y": 7,
-        "width": 25,
-        "height": 15
-      }
-    },
-    {
-      "id": 5811070951344460,
-      "layout": {
-        "x": 1,
-        "y": 81,
-        "width": 30,
-        "height": 8
-      },
-      "definition": {
-        "type": "note",
-        "content": "InnoDB is the storage management system for both MariaDB and MySQL.",
-        "background_color": "gray",
-        "font_size": "14",
-        "text_align": "left",
-        "show_tick": true,
-        "tick_pos": "50%",
-        "tick_edge": "right"
-      }
-    },
-    {
-      "id": 7403239476791868,
-      "layout": {
-        "x": 1,
-        "y": 55,
-        "width": 30,
-        "height": 25
-      },
-      "definition": {
-        "type": "note",
-        "content": "Most of the metrics and monitoring strategies outlined here also apply to MySQL-compatible technologies such MariaDB and Percona Server, with some notable differences. For instance, some of the features in the MySQL Workbench are not compatible with currently available versions of MariaDB.\n\nAmazon RDS users should check out our specialized monitoring guides for [MySQL on RDS](https://www.datadoghq.com/blog/monitoring-rds-mysql-performance-metrics/) and for the MySQL-compatible [Amazon Aurora](https://www.datadoghq.com/blog/monitoring-amazon-aurora-performance-metrics/).\n\n",
-        "background_color": "gray",
-        "font_size": "14",
-        "text_align": "left",
-        "show_tick": true,
-        "tick_pos": "50%",
-        "tick_edge": "right"
-      }
-    },
-    {
-      "id": 400903025136302,
-      "layout": {
-        "x": 177,
-        "y": 1,
-        "width": 47,
-        "height": 13
-      },
-      "definition": {
-        "type": "note",
-        "content": "The current rate of queries will naturally rise and fall, and as such it’s not always an actionable metric based on fixed thresholds. But it is worthwhile to alert on sudden changes in query volume—drastic drops in throughput, especially, can indicate a serious problem.",
-        "background_color": "gray",
-        "font_size": "14",
-        "text_align": "left",
-        "show_tick": true,
-        "tick_pos": "50%",
-        "tick_edge": "left"
-      }
-    },
-    {
-      "id": 1801107147992128,
-      "layout": {
-        "x": 1,
-        "y": 1,
-        "width": 30,
-        "height": 12
-      },
-      "definition": {
-        "type": "image",
-        "url": "/static/images/logos/mysql_large.svg",
```

Comment:
For preset dashboard, we should keep the integration logo as part of branding.

Reply:
Hello @ChristineTChen , thank you for your vigilance.
We are currently in the process of updating some integration dashboards to use the new dashboard layout, this process involve changing them from screenboard to this new layout.
We won't merge this to production until reviewed by product manager, but we will put the on staging for easier review.

Regarding logos, we are moving them in dashboard titles.

You can find more infos here: https://datadoghq.atlassian.net/browse/AI-1345

Reply:
I've updated the dashboard, it's ready for review.
As you mentioned last time, it's not a screenboard anymore, we will need to alter its type with the command after merging this